CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-mongodb ChangeLog
 
+## 8.2.1 - 2021-xx-xx
```

Comment:
Considering the extensive internal refactoring, perhaps 8.3.0 would be better. Leave some wiggle room in case something goes wrong. Maybe should also add a line item for the 'async' lib to async/await switch.

Reply:
Ok, updated.